content/no/es6.md

```diff
@@ -44,65 +43,59 @@ mest sannsynlig ekvivalent med tilstanden til
 
- * Block scoping
+*   Block scoping
 
-   * [let](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let)
+    *   [let](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let)
 
-   * [const](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const)
+    *   [const](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const)
 
-   * `function`-in-blocks
+    *   `function`-in-blocks
 
-> Fra og med v8 3.31.74.1, er block-scoped declarations [forsettlig implementert
-> med en ikke-kompatibel begrensning til strict modus
-> kode](https://groups.google.com/forum/#!topic/v8-users/3UXNCkAU8Es).
-> Utviklere bør være klar over at dette vil endres i takt med V8 sin
-> implementasjon av ES6-spesifikasjonen.
+    > Fra og med v8 3.31.74.1, er block-scoped declarations [forsettlig implementert med en ikke-kompatibel begrensning til strict modus kode](https://groups.google.com/forum/#!topic/v8-users/3UXNCkAU8Es). Utviklere bør være klar over at dette vil endres i takt med V8 sin implementasjon av ES6-spesifikasjonen.
 
- * Collections
+*   [Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) (bare i strict modus)
 
-   * [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)
+*   Collections
 
-   * [WeakMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap)
+    * [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)
 
-   * [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)
+    * [WeakMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap)
 
-   * [WeakSet](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet)
+    * [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)
 
- * [Generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*)
+    * [WeakSet](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakSet)
 
- * [Binary and Octal literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Numeric_literals)
+*   [Generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*)
 
- * [Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)
+*   [Binary and Octal literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Numeric_literals)
 
- * [New String methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla#Additions_to_the_String_object)
+*   [Object literal extensions](https://github.com/lukehoban/es6features#enhanced-object-literals) (shorthand properties and methods)
```

Comment:
Should this be translated as well? `(enkle egenskaper og metoder)`? Shorthand properties is a hard one to translate, due to shorthand being term lingo. I'd be fine with `(shorthand egenskaper og metoder)` as well.